editor/assets/default_renderpipeline/builtin-pipeline.ts

```diff
@@ -819,7 +817,9 @@ if (rendering) {
             for (let level = 0; level !== csmLevel; ++level) {
                 getCsmMainLightViewport(light, width, height, level, this._viewport, this._configs.screenSpaceSignY);
                 const queue = pass.addQueue(QueueHint.NONE, 'shadow-caster');
-                isWebGPU() || queue.setViewport(this._viewport);
+                if (!this._configs.isWebGPU) { // Temporary workaround for WebGPU
+                    queue.setViewport(this._viewport);
```

Comment:
Why is it not necessary to call `queue.setViewport` on webgpu platform?
What kind of problem will it cause? Any issue reference ?

Reply:
It will cause a bug in our backend implementation and is not related to WebGPU. GengineJS will fix it later.

Reply:
OK, got it.